src/test/java/elizCurtnazar/tests/tema8/pageObject/DisableDynamicButtonsPage.java

```diff
@@ -0,0 +1,78 @@
+package elizCurtnazar.tests.tema8.pageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+import java.util.List;
+
+public class DisableDynamicButtonsPage {
+
+    ChromeDriver driver;
+    WebDriverWait wait;
+    Wait<WebDriver> fluentWait;
+
+    public DisableDynamicButtonsPage(ChromeDriver driver) {
+        this.driver = driver;
+        //using WebDriverWait
+        wait = new WebDriverWait(driver, Duration.ofSeconds(9));
+        //using FluentWait
+        fluentWait = new FluentWait<WebDriver>(driver)
+                .withTimeout(Duration.ofSeconds(1))
+                .pollingEvery(Duration.ofSeconds(8));
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(id = "buttonmessage")
+    WebElement buttonMessage;
+
+    @FindBy(id = "button00")
+    WebElement startButton;
+
+    @FindBy(id = "button01")
+    WebElement buttonOne;
+
+    @FindBy(id = "button02")
+    WebElement buttonTwo;
+
+    @FindBy(id = "button03")
+    WebElement buttonThree;
+
+    public void clickButtons() {
+        Assert.assertEquals(getMessage(), "Click Buttons In Order");
```

Comment:
Please split this method into different methods, one per each buttons and move assertions from page object to test level. Thanks!

Reply:
done